src/new-client/src/app/lib/types/index.tsx

```diff
@@ -23,43 +21,80 @@ type AttributeValue = {
   value: string;
 };
 
-type User = {
+type Role = {
+  displayName: string;
   id: string;
-  roleAssignments: RoleAssignment[];
+  description: string;
+};
+
+type ApplyAvailabilityTemplateRequest = {
+  site: string;
+  from: string;
+  until: string;
+  template: AvailabilityTemplate;
+};
+
+type AvailabilityTemplate = {
+  days: DayOfWeek[];
+  sessions: AvailabilitySession[];
+};
+
+type DayOfWeek =
+  | 'Monday'
+  | 'Tuesday'
+  | 'Wednesday'
+  | 'Thursday'
+  | 'Friday'
+  | 'Saturday'
+  | 'Sunday';
+
+type AvailabilitySession = {
+  from: string;
+  until: string;
+  services: string[];
+  slotLength: number;
+  capacity: number;
 };
 
 type RoleAssignment = {
   scope: string;
   role: string;
 };
 
-type Role = {
-  displayName: string;
+type ServiceType = 'Covid' | 'Flu';
```

Comment:
Do we need to support RSV here?

Reply:
Ah, since removing the Availability Periods table on the pre-wizard page this type isn't actually used. I should probably remove it and re-implement it when it's needed (but yes, I imagine RSV should indeed be there!)